.idea/integreat-react-native-app.iml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="FacetManager">
+    <facet type="android" name="Android">
```

Comment:
This Java module seems weird.
My iml looks like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<module type="JAVA_MODULE" version="4">
  <component name="NewModuleRootManager" inherit-compiler-output="true">
    <exclude-output />
    <content url="file://$MODULE_DIR$">
      <sourceFolder url="file://$MODULE_DIR$/e2e" isTestSource="true" />
      <sourceFolder url="file://$MODULE_DIR$/flow-typed" isTestSource="false" />
      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
      <excludeFolder url="file://$MODULE_DIR$/android" />
      <excludeFolder url="file://$MODULE_DIR$/ios" />
    </content>
    <orderEntry type="inheritedJdk" />
    <orderEntry type="sourceFolder" forTests="false" />
  </component>
</module>
```

Reply:
The Java module is essentially only there for providing shortcuts to Android Tools such as AVD Manager and SDK Manager within Intellij...

Reply:
I adjusted the NewModuleRootManager component to adapt your suggestions there.